src/app/components/modules/Transfer.jsx

```diff
@@ -175,6 +182,42 @@ class TransferForm extends Component {
         const { submitting, valid, handleSubmit } = this.state.transfer;
         // const isMemoPrivate = memo && /^#/.test(memo.value); -- private memos are not supported yet
         const isMemoPrivate = false;
+
+        // Get names for the recent account transfers
+        const labelPreviousTransfers = tt(
+            'transfer_jsx.autocomplete_previous_transfers'
+        );
+        const labelFollowingUser = tt(
+            'transfer_jsx.autocomplete_user_following'
+        );
+
+        const transferToLog = currentAccount
+            .get('transfer_history')
+            .reduce((acc, cur) => {
+                if (cur.getIn([1, 'op', 0]) === 'transfer') {
+                    const username = cur.getIn([1, 'op', 1, 'to']);
+                    const numTransfers = acc.get(username)
+                        ? acc.get(username).numTransfers + 1
+                        : 1;
+                    return acc.set(username, {
+                        username,
+                        label: `${numTransfers} ${labelPreviousTransfers}`,
+                        numTransfers,
+                    });
+                }
+                return acc;
+            }, Map());
+
+        // Build a combined list of users you follow & have previously transferred to,
+        // and sort it by the number of previous transfers.
+        const autocompleteUsers = this.props.following
+            .toMap()
+            .map(username => ({ username, label: labelFollowingUser }))
+            .merge(transferToLog)
+            .sortBy(u => u.numTransfers)
```

Comment:
@bnchdrff @netuoso - this sort doesn't work.

Reply:
are you saying we shouldn't sort by the number of previous transfers, or that it actually doesn't work? it's working for me:

![screenshot from 2017-12-17 13-37-36](https://user-images.githubusercontent.com/99194/34082606-78876d8c-e32f-11e7-9358-3c6dd5045d48.png)

Reply:
ah found a condition where it doesn't work, fixing it now...

Reply:
It didn't sort at all on live data. `.sortBy` return(s/ed) a list that has the same order as `this.props.following`. 

There were 2 transfers in the list generated by `.merge(transferToLog)` with the rest being undefined. Neither of those had their order changed. The comparator fn will prob need to move to an `a > b? 1 : (a < b? : -1 : 0)` operation.